resources/validations/src/ssp/policy-and-procedure.sch

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema
```

Comment:
When these are added as to ssp.sch (at the bottom here: https://github.com/18F/fedramp-automation/blob/SSP-cleanup/resources/validations/src/ssp.sch#L505-L506), I don't think it will accept a nested `sch:schema`. We may need to remove the containing `sch:schema` here, and just put a pattern in these include files.